staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -340,6 +365,12 @@ loop:
 				utilruntime.HandleError(fmt.Errorf("%s: expected type %v, but watch event object had type %v", r.name, e, a))
 				continue
 			}
+			if r.expectedGVK != nil {
```

Comment:
would format the block above the same way:
```
if r.expectedType != nil {
   ...
}

Reply:
Done